.travis.yml

```diff
@@ -4,5 +4,10 @@ jdk:
 before_install:
   - "echo $JAVA_OPTS"
   - "export JAVA_OPTS=-Xmx512m"
+
+script:
+  - mvn javadoc:jar
+  - mvn javadoc:test-aggregate
```

Comment:
I think by adding explicit options here for "script" we lose the default behavior and now no longer actually run the tests.